help/destinations/catalog/streaming/tiktok.md

```diff
@@ -0,0 +1,114 @@
+
+### Authenticate to destination {#authenticate}
+
+To authenticate to the destination, you will be redirected to login to your TikTok Ads Manager account and authorize Adobe to manage audiences on your behalf. 
+
+![TikTok permission selection](/help/destinations/destination-sdk/docs-framework/assets/tiktok-authenticate-destination.png "Image of TikTok UI for selecting permissions")
+
+### Fill in destination details {#destination-details}
+
+To configure details for the destination, fill in the required and optional fields below. An asterisk next to a field in the UI indicates that the field is required.
+
+![Destination connection details](/help/destinations/destination-sdk/docs-framework/assets/tiktok-configure-destination-details.png "Image of the Platform UI, showing destination connection details to be filled in")
+
+*  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
+*  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+*  **[!UICONTROL TikTok Ads Manager ID]**: Your TikTok Ads Manager ID.
+
+### Enable alerts {#enable-alerts}
+
+You can enable alerts to receive notifications on the status of the dataflow to your destination. Select an alert from the list to subscribe to receive notifications on the status of your dataflow. For more information on alerts, see the guide on [subscribing to destinations alerts using the UI](../../ui/alerts.md).
+
+When you are finished providing details for your destination connection, select **[!UICONTROL Next]**.
+
+## Activate segments to this destination {#activate}
+
+>[!IMPORTANT]
+> 
+>To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+Read [Activate profiles and segments to streaming segment export destinations](/help/destinations/ui/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+### Map identities {#map}
+
+Below is an example of correct identity mapping when exporting segments to TikTok Ads Manager.
+
+Selecting source fields:
+
+* Select an identifier (For example: Email_LC_SHA256) as source identity that uniquely identifies a profile in Adobe Experience Platform and TikTok Ads Manager.
+
+Selecting target fields:
+
+* Select the email namespace as target identity.
+
+![Identity mapping](/help/destinations/destination-sdk/docs-framework/assets/tiktok-map-identity.png "Image of the Platform UI, mapping of identities")
```

Comment:
Please update the image link to the Adobe-supported format: 

![Image of the Platform UI, mapping of identities.](/help/destinations/destination-sdk/docs-framework/assets/tiktok-map-identity.png)

Reply:
Could you clarify what the issue here is?

Reply:
I actually need to take back this comment. I got confused but the format you used is absolutely correct (as per [these instructions](https://experienceleague.adobe.com/docs/contributor/contributor-guide/writing-essentials/markdown.html?lang=en#images)).